streampipes-client-python/streampipes_client/client/client.py

```diff
@@ -102,6 +102,7 @@ def __init__(
         # provide all available endpoints here
         # name of the endpoint needs to be consistent with the Java client
         self.dataLakeMeasureApi = DataLakeMeasureEndpoint(parent_client=self)
+        self.dataStreamApi = SpDataStreamEndpoint(parent_client=self)
```

Comment:
@SvenO3 would you mind just calling the class `DataStreamEndpoint`? (same applies for `SpDataStream`) 
I'm more in favor of class names without this prefix, since we would then need to add it to almost every class.
What do you think?

Reply:
Yes, we can change this. I just used the same names as in the StreamPipes API and this prefix is only added to the SpDataStream and SpDataSet there